schema/workflow.json

```diff
@@ -1188,8 +1185,8 @@
           "description": "Data condition name"
         },
         "condition": {
-          "type": "string",
-          "description": "Workflow expression evaluated against state data. Must evaluate to true or false"
+          "$ref": "common.json#/definitions/booleanOrMarkup",
+          "description": "Boolean or workflow expression that evaluates to boolean against state data"
```

Comment:
why "boolean or workflow expression" you would never just put "false" or "true" as the value here. its always an expression which must return boolean value

Reply:
Have you ever altered your code to `if ( true )` or `if ( false )` for testing? Maybe it's bad style, but I thought that only the result counts, i.e. whatever is in there should eventually be boolean.